app/Model/RatingComment.php

```diff
@@ -8,4 +8,24 @@
 class RatingComment extends Model
 {
     use SoftDeletes;
+    
+    /**
+     * Relationship with users model
+     *
+     * @return \App\Model
+     */
+    public function users()
```

Comment:
`RatingComment` belongs to `User` nên tại sao lại dùng số nhiều trong trường hợp này?